src/shogun/features/StringFeatures.cpp

```diff
@@ -1730,12 +1730,17 @@ template<class ST> void CStringFeatures<ST>::init()
 	original_num_symbols=0;
 
 	m_parameters->add((CSGObject**) &alphabet, "alphabet");
+
 	m_parameters->add_vector(&features, &num_vectors, "features",
```

Comment:
every raw array, which was previously registered with `add_vector` should also be registered with `watch_param`